packages/gatsby/src/query/utils.ts

```diff
@@ -1,3 +1,5 @@
+import { Path } from "graphql/jsutils/Path"
```

Comment:
Looks dangerous 😬 importing from an internal

Reply:
That's a type.

Reply:
OK nvm me than 👍